gears/src/application/keepers/params.rs

```diff
@@ -16,6 +18,17 @@ pub trait ParamsKeeper<PSK: ParamsSubspaceKey> {
 
     fn psk(&self) -> &PSK;
 
+    fn check_key<SL: AsRef<[u8]>>(key: SL) -> bool {
+        <Self::Param as ParamsSerialize>::keys()
+            .iter()
+            .map(|this| this.as_bytes())
+            .collect::<HashSet<_>>()
+            .contains(key.as_ref())
+    }
+
+    #[cfg(all(feature = "governance"))]
+    fn validate(key: impl AsRef<[u8]>, value: impl AsRef<[u8]>) -> bool;
```

Comment:
How it can resolve the type behind the bytes? Is it required?

Reply:
Developer resolves